script/bulk_import/base.rb

```diff
@@ -538,7 +541,18 @@ def random_email
   end
 
   def pre_cook(raw)
-    cooked = @markdown.render(raw).scrub.strip
+    cooked = raw
+
+    # Convert YouTube URLs to lazyYT DOMs before being transformed into links
+    cooked.gsub!(/\nhttps\:\/\/www.youtube.com\/watch\?v=(\w+)\n/m) do
```

Comment:
The `/m` modifier is for "multiline" ;)

The `gsub` is taking care of the "several youtube links"